packages/atlas/src/config/categories.tsx

```diff
@@ -1,113 +1,140 @@
-import { ReactNode } from 'react'
+import { concat, intersection } from 'lodash-es'
 
-import {
-  SvgCategoriesAutosAndVehicles,
-  SvgCategoriesComedy,
-  SvgCategoriesEducation,
-  SvgCategoriesEntertainment,
-  SvgCategoriesFilmAndAnimation,
-  SvgCategoriesGaming,
-  SvgCategoriesHowtoAndStyle,
-  SvgCategoriesMusic,
-  SvgCategoriesNewsAndPolitics,
-  SvgCategoriesNonprofitsAndActivism,
-  SvgCategoriesPeopleAndBlogs,
-  SvgCategoriesPetsAndAnimals,
-  SvgCategoriesScienceAndTechnology,
-  SvgCategoriesSports,
-  SvgCategoriesTravelAndEvents,
-} from '@/components/_icons'
+import { VideoId } from '@/joystream-lib/types'
 
-export type VideoCategoryData = { id: string; icon: ReactNode; color: string; coverImg: string }
-export const videoCategories: Record<string, VideoCategoryData> = {
-  '14': {
-    id: '14',
-    icon: <SvgCategoriesScienceAndTechnology />,
+export type VideoCategoryData = {
```

Comment:
Maybe just `DisplayCategory`? We should try to differentiate between different types of categories to make it clear which ones we're using where